### ISSUE TITLE
UX: Make connection between "Documentation" and the actual errors more clear

### ISSUE THREAD

**Issue by staabm** [sonarwhal/sonarwhal.com#320]
> I just scanned a webproject we made and I can see a lot of errors like
> ![grafik](https://user-images.githubusercontent.com/120441/33420461-f1e8e7fc-d5ae-11e7-984f-5ffa72361907.png)
> this makes me think of why is the `x-xss-protection header not needed`? in the first place its a security header and its not obvious why I should not use them (or maybe its a more complex context here which I dont know about and needs explanation?)

origin issue: https://github.com/sonarwhal/sonarwhal/issues/682#issuecomment-348187171

**Comment by staabm** [comment]
ohh wow, its such a huge button and my brain wasnt able to get the connection between each error below and this button... I guess this means we should do something about it.

Proposal:
![grafik](https://user-images.githubusercontent.com/120441/33435650-01db083c-d5e3-11e7-9d32-8721b623775e.png)

I guess things would be easier when the "docu" symbol would appear when hovering with the mouse over a error-msg. that way the user has a indicator that there is some info available which is directly related to the error being shown.

**Comment by staabm** [comment]
especially for error messages which are far more away from the "Documentation" button, the benefit is getting obvious

![grafik](https://user-images.githubusercontent.com/120441/33435801-6a2e7e6e-d5e3-11e7-8634-482c1caf55d2.png)

**Comment by alrra** [comment]
@ststimac @sonarwhal/core Thoughts?

**Comment by ststimac** [comment]
I don't think we should add an icon to every error. Some people have 200+ errors and that will just be a bit redundant. 

We can try making the rules themself a link, but there was also a suggestion to add another call to action to take users to the documentation at the bottom of the results, which I think would be helpful for when some rules have a long list of errors and the documentation button disappears from view as users scroll down.

**Comment by alrra** [comment]
> I don't think we should add an icon to every error. 

I agree.

> We can try making the rules themself a link,

:+1:

Just so that there is no confusion, we are referring to also making the following parts point to the documentation:

![](https://user-images.githubusercontent.com/1223565/33498289-9f268506-d685-11e7-9dca-626009377ba5.png)

----

@staabm Thoughts? Do you think that would make it more clear / easy for new users such as yourself?

**Comment by staabm** [comment]
Depends on the styles applied to this headline I guess
